doc/passw_hardening/sonic-passwh.yang

```diff
@@ -0,0 +1,91 @@
+module sonic-passwh {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-passwh";
+	prefix passwh;
+
+    description "PASSWORD HARDENING YANG Module for SONiC OS";
+
+	revision 2021-10-12 {
+        description
+            "First Revision";
+    }
+    typedef feature_state {

```

Comment:
Please follow sonic yang model guideline, every definition should inside top level container, in this case, should inside sonic-passwh